### PR TITLE
filters for erc20 tokens

### PR DESCRIPTION
**Pull request by IAmMorrow**
image

### Type

Feature

### Context

https://github.com/LedgerHQ/ledger-live-mobile/pull/931

### Parts of the app affected / Test plan

Receive / Send account select screen

### CODE REVIEW

File: src/helpers/formatAccountSearchResults.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  TokenAccount,
+} from "@ledgerhq/live-common/lib/types/account";
+
+export type SearchResult = {
+  account: Account | TokenAccount,
+  match?: boolean,
+};
+
+const flattenStructuredSearchResults = structuredResults =>
+  reduce(
```
Comment (gre) [inline src/helpers/formatAccountSearchResults.js]: we can directly use `.reduce` on the JS Array

**Comment by gre** [comment]
there is something weird in this PR that contains stuff done from other commits, could you try to rebase from topic/erc20 (or cherry-pick if it's one commit)?

**Comment by gre** [comment]
let's merge this and we can test on topic/erc20

**Comment by IAmMorrow** [comment]
Fixed it :D

**Pull request merged**